three.js/src/threex/threex-artoolkitcontext-nft.js

```diff
@@ -18,7 +18,7 @@ ARjs.Context = THREEx.ArToolkitContext = function (parameters, sourceParameters)
         matrixCodeType: '3x3',
 
         // url of the camera parameters
-        cameraParametersUrl: ARjs.Context.baseURL + 'parameters/camera_para.dat',
```

Comment:
as written in the https://github.com/AR-js-org/AR.js/issues/44 this was wrong and wasn't possible to set the default path

Reply:
so it's now solved right? I can dismiss this comment?

Reply:
yes we can close this.